Assets/TabToggleParent.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System.Collections;
```

Comment:
If this is a finding class (which I believe it is), could you add a comment right above it to say so? I don't normally but now seeing this I think I should